web/portal/src/main/resources/locale/portal/webui_en.properties

```diff
@@ -649,16 +649,7 @@ UIPageTemplateOptions.UIDropDown.label.tabsPageConfigs=Tabs
 ##################################################################
 UIWizardPageSelectLayoutForm.tab.label.UIPageTemplateOptions=Page template options
 UIWizardPageSelectLayoutForm.label.normalPage.EmptyLayout=Blank Page
-UIWizardPageSelectLayoutForm.label.normalPage.CurrentLayout=Current Layout
```

Comment:
- verify if this label is needed
- we should clean the other properties

Reply:
cleaned properties !